taipy/core/_manager/_manager.py

```diff
@@ -59,6 +59,13 @@ def _get_all(cls) -> List[EntityType]:
         """
         return cls._repository._load_all()
 
+    @classmethod
+    def _get_all_by(cls, by) -> List[EntityType]:
+        """
+        Returns all entities.
+        """
+        return cls._repository._load_all_by(by)
+
```

Comment:
If we add a new method in the managers, we need to authorize it in enterprise version.

Reply:
Ok, we have to do it.

Reply:
@florian can you override this method on enterprise version?